.github/workflows/deploy.yml

```diff
@@ -1,30 +1,24 @@
 name: Deploy
-
 concurrency:
-  group: ${{ github.event.repository.name }}-deploy
+  group: deploy
   cancel-in-progress: true
-
 on:
   workflow_dispatch:
   push:
     branches:
       - main
-
 jobs:
-  continuous-integration:
-    uses: ./.github/workflows/continuous-integration.yml
-    name: continuous integration
-
+  CI:
+    uses: ./.github/workflows/ci.yml
```

Comment:
Losing the name here seems like a step backwards? ~I presume this is coming from the base template.~ Yep, I've put some notes on the base ticket.

Reply:
Yep, changed this in the cookiecutter